<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingPanel</class>
 <widget class="QWidget" name="SettingPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>221</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>背景音乐</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="openBgmBtn">
      <property name="text">
       <string>开</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="closeBgmBtn">
      <property name="text">
       <string>关</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>音效</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="soundsCombox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="resetSettingBtn">
      <property name="text">
       <string>恢复默认设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="returnBtn">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
